src/test/java/com/softserveinc/ita/FacultiesTest.java

```diff
@@ -22,24 +21,26 @@ public void openFacultiesPage() {
     }
 
     @Test(dataProvider = "searchValues")
-    public void searchFieldWorksWithValidInput(String value, String expectedValue) {
+    public void verifySearchFieldWorksWithValidInput(String value, String expectedValue) {
```

Comment:
synchronize your local "main" branch with remote and after this merge into your current branch